tests/controls/windowtest.cpp

```diff
@@ -459,3 +460,81 @@ TEST_CASE_METHOD(WindowTestCase, "Window::SizerErrors", "[window][sizer][error]"
 
     REQUIRE_NOTHROW( delete child );
 }
+
+TEST_CASE_METHOD(WindowTestCase, "Window::Refresh", "[window]")
+{
+    wxWindow* const parent = m_window;
+    wxWindow* const child1 = new wxWindow(parent, wxID_ANY, wxPoint(10, 20), wxSize(80, 50));
+    wxWindow* const child2 = new wxWindow(parent, wxID_ANY, wxPoint(110, 20), wxSize(80, 50));
+    wxWindow* const child3 = new wxWindow(parent, wxID_ANY, wxPoint(210, 20), wxSize(80, 50));
+
+    m_window->SetSize(300, 100);
+
+#if wxDEBUG_LEVEL
```

Comment:
Any reason we couldn't always do this? AFAICS it doesn't do any harm in non-debug build build too, does it?

Reply:
Removed now